css/mail.scss

```diff
@@ -522,6 +536,14 @@ input.submit-message,
 	display: inline-block;
 }
 
+.mail-message-actions-menu{
```

Comment:
cc @skjnldsv regarding the need for this special rule or if we can somehow streamline our open/closed popover states somehow.

Reply:
Yup, @kevgathuku take a look at https://docs.nextcloud.com/server/14/developer_manual/design/list.html :)

Reply:
Thanks for the pointers. This should be fixed now.